examples/files_example/public/components/app.tsx

```diff
@@ -130,22 +148,135 @@ export const FilesExampleApp = ({ files, notifications }: FilesExampleAppDeps) =
     },
   ];
 
+  return (
+    <>
+      <EuiTitle size="s">
+        <h2>{title}</h2>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      {!!description && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiText>{description}</EuiText>
+        </>
+      )}
+      <EuiInMemoryTable
+        columns={columns}
+        items={items}
+        itemId="id"
+        loading={isLoading || isDeletingFile}
+        error={error ? JSON.stringify(error) : undefined}
+        sorting
+        search={{
+          toolsRight: renderToolsRight(),
+        }}
+        pagination
+      />
+    </>
+  );
+};
+
+export const FilesExampleApp = ({ files, notifications }: FilesExampleAppDeps) => {
```

Comment:
I suggest not keeping the changes to this example. It is nice to have an example that includes just basic functionality and minimal code needed to get things working, without additional specific abstractions that are harder to read and copy-paste.

I think we should leave a basic example with a single table and if we want we can create a separate sub-example for this additional management config.

But IMO just documentation/typescript types comments for this config should be enough